buildpacks/dotnet/src/main.rs

```diff
@@ -79,7 +79,17 @@ impl Buildpack for DotnetBuildpack {
         let started = std::time::Instant::now();
         print::bullet("SDK version detection");
 
-        let solution = get_solution_to_publish(&context.app_dir)?;
+        let solution = buildpack_configuration.solution_file.map_or_else(
+            || get_solution_to_publish(&context.app_dir),
+            |path| {
+                print::sub_bullet(format!(
```

Comment:
Nitpick and in no way blocking, so feel free to ignore: 

Ideally, we'd not side-effect in mapping functions and IMO and `if let/else` might be better.